src/main/scala-2.11/markets/engines/CDAMatchingEngine.scala

```diff
@@ -59,6 +43,11 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
     immutable.TreeSet.empty[BidOrder](bidOrdering) ++ _bidOrderBook
   }
 
+  /** Find a match for an incoming order.
+    *
+    * @param incoming the order to be matched.
+    * @return a collection of matches.
+    */
   def findMatch(incoming: Order): Option[immutable.Queue[Matching]] = {
 
     incoming match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1834547231/issues/source?bid=2921361&fileBranchId=3080901#l53)